deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1291,20 +1292,28 @@ def callback_registry_count_entries(line):
         return match.group(1)
 
 
+def callback_key_event(line):
+    event = callback_detect_event(line)
+
+    if event is None:
+        return None
+
+    if event['data']['attributes']['type'] != 'registry_key' or event['data']['path'] == registry_ignore_path:
+        return None
+
+    return event
```

Comment:
```suggestion
event = callback_detect_event(line)

match_event = None if event is None or event['data']['attributes']['type'] != 'registry_key' \
    or event['data']['path'] == registry_ignore_path else event
return match_event
```

Reply:
Instead of a long one-liner I merged both if conditions, think it might be easier to read.
[Commit](https://github.com/wazuh/wazuh-qa/pull/1670/commits/c5f646dba669a1facacd74025ba515b4b452bda6)